node/src/BaseClient.ts

```diff
@@ -3330,6 +3332,63 @@ export class BaseClient {
         return this.createWritePromise(createObjectRefcount(key));
     }
 
+    /**
+     * Invokes a previously loaded function.
+     *
+     * See https://valkey.io/commands/fcall/ for more details.
+     *
+     * since Valkey version 7.0.0.
+     *
+     * @remarks When in cluster mode, all `keys` must map to the same hash slot.
+     * @param func - The function name.
+     * @param keys - A list of `keys` accessed by the function. To ensure the correct execution of functions,
+     * all names of keys that a function accesses must be explicitly provided as `keys`.
+     * @param args - A list of `function` arguments and it should not represent names of keys.
+     * @returns The invoked function's return value.
+     *
+     * @example
+     * ```typescript
+     * const response = await client.fcall("Deep_Thought");
+     * console.log(response); // Output: Returns the function's return value.
+     * ```
+     */
+    public fcall(
+        func: string,
+        keys?: string[],
+        args?: string[],
```

Comment:
Maybe
```suggestion
        keys: string[],
        args: string[],
```
So if a user wants to skip `keys`, API call will be `fcall(func, [], [...])`